server/controllers/ArticleController.js

```diff
@@ -1,4 +1,4 @@
-import { Boolify } from 'node-boolify';
+import { Sequelize } from 'sequelize';
```

Comment:
Hey Timi,
The Sequelize import is rather redundant as it is also part of `db` you imported from `../models`

Reply:
Thanks for the feedback, I have refactored it.